src/styles/variables.ts

```diff
@@ -43,6 +43,7 @@ export default {
     avatarSizeMentionIcon: 16,
     avatarSizeSmallSubscript: 14,
     defaultAvatarPreviewSize: 360,
+    fabBottom: 25,
```

Comment:
What does this mean exactly? It's the distance of the FAB to the bottom of the screen? What was it before we made any of these changes?

Reply:
Yes, it's this distance. I've added this variable only to use it in a calculation of the menu position. Thanks to that it's easier to understand how we set the menu position

Reply:
Sounds good.